packages/workspace/src/command-line/nx-commands.ts

```diff
@@ -279,6 +279,13 @@ function withAffectedOptions(yargs: yargs.Argv): yargs.Argv {
     .option('verbose', {
       describe: 'Print additional error stack trace on failure',
     })
+    .option('excludeTags', {
```

Comment:
I actually had that in one of my prior commits but wasn't sure if it was fully necessary as it affected some tests (because of the default value) that weren't really relevant (ex. some tests failed cause it was expecting the default for excludeTags but it didn't have a default for other params that accepted a csv list).

Instead, I opted for checking to see if it was an array in the code when it is being used.

Not sure what the best way to go about it is. 

Would like to hear what one of the Nx team members thinks.

Reply:
Moving it close to excludes isn't a bad idea though. Sorry i missed that my first read.